.github/workflows/review.yaml

```
@@ -17,25 +20,35 @@ jobs:
-    #     restore-keys: |
-    #       ${{ runner.os }}-ffmpeg-
+    - name: build x264
+      run: |
+        git clone --branch stable --depth 1 https://code.videolan.org/videolan/x264.git
+        cd x264
+        ./configure --enable-shared
+        make -j $2
+        make install
+
+    - name: build x265
+      run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:2:4: Double quote to prevent globbing and word splitting

(shellcheck)